### PR TITLE
add parameter CHANNELS_CONFIG

### PR DESCRIPTION
**Pull request by DeadByElpy**

### CODE REVIEW

File: tests/commands.test.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import * as message from '../src/utils/message.js';
 import getBytesFromHex from '../src/utils/getBytesFromHex.js';
 import getHexFromBytes from '../src/utils/getHexFromBytes.js';
 import * as hardwareTypes from '../src/constants/hardwareTypes.js';
-import * as deviceParameters from '../src/constants/deviceParameters.js';
+import * as deviceParameters from '../src/constants/deviceParameters/index.js';
```
Comment (DarkPark) [inline tests/commands.test.ts:9]:
```suggestion
```

File: src/constants/deviceParameters/index.ts [hunk 2]
```diff
@@ -46,3 +49,7 @@ export const ABSOLUTE_DATA_MULTI_CHANNEL = 29;
 
 /** enable absolute value sending for a specific channel */
 export const ABSOLUTE_DATA_ENABLE_MULTI_CHANNEL = 30;
+
+export * as rx2SpreadFactors from './rx2SpreadFactors.js';
```
Comment (DarkPark) [inline src/constants/deviceParameters/index.ts:53]: this shouldn't be here

File: src/constants/deviceParameters/channelsConfigs.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
+/**
+ * Possible values of channels config
+ *
+ * @packageDocumentation
+ */
+
+export const CHANNELS_ENABLED_1_2_3_4_DISABLED_SERIAL = 0;
```
Comment (DarkPark) [inline src/constants/deviceParameters/channelsConfigs.ts:7]: do we really need these constants? they look terrible.

how about info table in jooby-docs and a link here to docs?

File: tests/commands.test.ts (hunk 1 above)
Comment (DeadByElpy) [reply tests/commands.test.ts:9]: all unused imports removed

File: src/constants/deviceParameters/index.ts (hunk 2 above)
Comment (DeadByElpy) [reply src/constants/deviceParameters/index.ts:53]: in that case path to constants `jooby-codec- constants/deviceParameters/rx2SpreadFactors` will be unavailable, any proposal how to do it?

File: src/constants/deviceParameters/channelsConfigs.ts (hunk 3 above)
Comment (DeadByElpy) [reply src/constants/deviceParameters/channelsConfigs.ts:7]: constants - comfortable, names - representation of that complex format, and no idea how to keep it in constants without these names

File: src/constants/deviceParameters/index.ts (hunk 2 above)
Comment (DeadByElpy) [reply src/constants/deviceParameters/index.ts:53]: reworked

File: src/constants/deviceParameters/channelsConfigs.ts (hunk 3 above)
Comment (DeadByElpy) [reply src/constants/deviceParameters/channelsConfigs.ts:7]: constants removed

**Review by DarkPark — APPROVED** [review]

**Pull request merged**